packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+| orientation | `vertical`, `horizontal`                                                             | `vertical` | Orientation of the card                                                                   |
+| size        | `smallest`, `smaller`, `small`, `medium`, `large`                                    | `medium`   | Define the minimum size of the card. Smaller sizes only apply to horizontal card          |
+| scale       | `fixed`, `auto-width`, `auto-height`, `auto`, `fluid-width`, `fluid-height`, `fluid` | `auto`     | Manages how the card handles it's scaling depending on the content                        |
+| onClick     | function                                                                             | undefined  | Function to call when clicking the card. Makes the card assume the interactive appearance |
+| appearance  | `filled`, `filled-alternative`, `outline`, `subtle`                                  | `filled`   | Define the appearance of the card                                                         |
+| selectable  | boolean                                                                              | false      | Makes the card selectable by adding a checkbox to the _Actions_ area                      |
+| selected    | boolean                                                                              | false      | Set to `true` if card is selected                                                         |
+| expandable  | boolean                                                                              | false      | Allow card to expand to show whole content                                                |
+| context     | _slot_                                                                               | undefined  | Slot for the context menu, when right-clicking the card                                   |
+| disabled    | boolean                                                                              | false      | Makes the card disabled                                                                   |
+
+#### `scale` property
+
+- `fixed`: Size of the card is fixed.
+- `auto-width`: `width` is set to `fit-content`.
+- `auto-height`: `height` is set to `fit-content`.
+- `auto`: `width` and `height` are set to `fit-content`.
+- `fluid-width`: `width` is set to `100%`.
+- `fluid-height`: `height` is set to `100%`.
+- `fluid`: `width` and `height` are set to `100%`.
+
+### CardHeader
+
+#### Anatomy
+
+![visual anatomy of the CardHeader component](./assets/CardHeader.png)
+
+#### API
+
+| Property    | Values | Default   | Purpose                                           |
+| ----------- | ------ | --------- | ------------------------------------------------- |
+| image       | _slot_ | undefined | Logo or image to better contextualize the content |
+| title       | _slot_ | undefined | Main title of the card                            |
```

Comment:
`title` collides with HTML `title` attribute name, so most probably we should find a different name, for example header

Reply:
Good point, changed to `header`